features/steps/files_publisher_component.go

```
@@ -101,3 +91,19 @@ func (c *FilePublisherComponent) Close() error {
 	}
 	return nil
 }
+
+func deleteObjectsInBucket(bucketName string, client *s3.Client) {
+	listObjectInput := &s3.ListObjectsInput{
+		Bucket: aws.String(bucketName),
+	}
+
+	listObjectOutput, _ := client.ListObjects(context.Background(), listObjectInput)
+
+	for _, object := range listObjectOutput.Contents {
+		deleteObjectInput := &s3.DeleteObjectInput{
+			Bucket: aws.String(bucketName),
+			Key:    object.Key,
+		}
+		_, _ = client.DeleteObject(context.Background(), deleteObjectInput)
```

should we ignore this error?